shared/version/api.go

```diff
@@ -372,6 +372,7 @@ var APIExtensions = []string{
 	"ovn_nic_acceleration_vdpa",
 	"cluster_healing",
 	"instances_state_total",
+	"instances_rebuild",
```

Comment:
Adding an API extension should be a separate commit please, see other API extensions for examples.